WISE-Community/WISE-Client
# feat(VLE): Tabbed Navigation Theme

## Changes
- Add new "tab" theme for the VLE

## Test Prep
- Add ```"theme": "tab"``` as top-level key/value in project.json

## Test
- Default theme: Preview/Student works as before
- Tabbed theme
   - tabs appear at the top of the screen, one for each lesson in the unit
      - click on a tab. It should take you to the first step in that lesson
   - prev/next buttons appear at the bottom of the page after all the components
      - they are disabled if there is no prev/next, including when you can't go next because of constraint 
      - click on prev/next should take you to prev/next step in the unit. 
         - it should always scroll to the top of the page after navigating to the new step 
         - the selected tab should reflect the lesson that the current step is in. 

Closes #788
